src/util/scrollbar.ts

```diff
@@ -0,0 +1,44 @@
+import {Injectable, Inject} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+
+
+@Injectable()
+export class ScrollBar {
+  constructor(@Inject(DOCUMENT) private _document) {}
+
+  compensate() {
```

Comment:
Could you please add explicit return type so it is easier to read? We've got a `return` statement in this function but it is not immediately clear what is returned.

Reply:
Yes, I'll add it along with documentation about it